recipes/qhull/all/patches/fix-cmake-7.3.2.patch

```diff
@@ -22,14 +22,14 @@
 -    set(qhull_STATICR qhullstatic_r)
 -endif()
 +include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
- 
++
 +set(qhull_CPP qhullcpp)
```

Comment:
does it really need a patch? can we set these `qhull_*` variables from the outside (via `cmake -Dqhull_CPP=whatver ...`)?

Reply:
this part of the patch for old versions is not new. it is required.